examples/03-Cyclic-Examples/05-modal_cyclic_no_expansion.py

```diff
@@ -0,0 +1,116 @@
+# noqa: D400
+"""
+.. _ref_basic_cyclic:
+
+Get base and duplicate sectors (real and imaginary) results for modal cyclic symmetry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to extract results from a modal cyclic symmetry model.
+
+"""
+
+from ansys.dpf import post
+from ansys.dpf.post import examples
+
+###############################################################################
+# Perform required imports
+# ------------------------
+# This example uses a supplied file that you can
+# get using the ``examples`` module.
+
+
+###############################################################################
+# Get ``Simulation`` object
+# -------------------------
+# Get the ``Simulation`` object that allows access to the result. The ``Simulation``
+# object must be instantiated with the path for the result file. For example,
+# ``"C:/Users/user/my_result.rst"`` on Windows or ``"/home/user/my_result.rst"``
+# on Linux.
+
+example_path = examples.download_modal_cyclic()
+simulation = post.load_simulation(example_path)
+
+# for no autocompletion, this line is equivalent to:
+simulation = post.ModalMechanicalSimulation(example_path)
+
+# print the simulation to get an overview of what's available
+print(simulation)
+
+
+###############################################################################
+# Get base and duplicate sectors displacement results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# With ``expand_cyclic=False``, the result providers (stress, displacement, and so on) return
+# results for base and duplicate sectors for a cyclic symmetry model.
+
+# Create displacement operator
+u_cyc = simulation.displacement(all_sets=True, expand_cyclic=False)
+
+###############################################################################
+# The output dataframe print displays the organization of the data.
+# The label "base_sector" gives access to base sectors
+# results with base_sector=1 and duplicate sector with base_sector=0 for all
```

Comment:
@cbellot000 base_sector label is a boolean?

Reply:
```suggestion
# results with base_sector=True and duplicate sector with base_sector=False for all
```